jobs/redis/templates/config/redis.conf.erb

```diff
@@ -14,11 +14,11 @@
 
 # By default Redis does not run as a daemon. Use 'yes' if you need it.
 # Note that Redis will write a pid file in /var/run/redis.pid when daemonized.
-daemonize yes
+daemonize no
 
 # When running daemonized, Redis writes a pid file in /var/run/redis.pid by
 # default. You can specify a custom pid file location here.
-pidfile /var/vcap/sys/run/redis/redis.pid
+# pidfile /var/vcap/sys/run/redis/redis.pid
```

Comment:
Can this be removed entirely now that `daemonize no` is enabled? I find it counter-intuitive that this works because `/var/vcap/sys/run` should not be writable inside a bpm container.